uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlement.php

```diff
@@ -647,4 +638,22 @@ public static function getBreakdown($params) {
     return $results;
   }
 
+  /**
+   * Returns LeaveBalanceChange Options for Type ID in a key-value pair
```

Comment:
Just by reading the description of this method I can't understand what this method does. What is the key and what is the value for each pair?